### ISSUE TITLE
USB printer

### ISSUE THREAD

**Issue by DebianC** [python-escpos/python-escpos#278]
Hello, thanks for seeing my question. I tried this

```

from escpos.printer import Usb

p=Usb(0x0fe6,0x811e,0,0x81,0x01)
p.text(“helloworldandyou”)
p.cut（）
```

and got errors as bellow.It seems that my printer is not connected, could you please tell me a possible way to solve this? Thank you very much!

### Bug description

```
pi@raspberrypi:~/pyusb $ python trythis2.py
Traceback (most recent call last):
  File "trythis2.py", line 4, in <module>
    p=Usb(0x0fe6,0x811e,0,in_ep=0x81,out_ep=0x01)
  File "/usr/local/lib/python2.7/dist-packages/escpos/printer.py", line 51, in __init__
    self.open()
  File "/usr/local/lib/python2.7/dist-packages/escpos/printer.py", line 62, in open
    check_driver = self.device.is_kernel_driver_active(0)
  File "/home/pi/pyusb/usb/core.py", line 1061, in is_kernel_driver_active
    self._ctx.managed_open()
  File "/home/pi/pyusb/usb/core.py", line 102, in wrapper
    return f(self, *args, **kwargs)
  File "/home/pi/pyusb/usb/core.py", line 120, in managed_open
    self.handle = self.backend.open_device(self.dev)
  File "/home/pi/pyusb/usb/backend/libusb1.py", line 786, in open_device
    return _DeviceHandle(dev)
  File "/home/pi/pyusb/usb/backend/libusb1.py", line 643, in __init__
    _check(_lib.libusb_open(self.devid, byref(self.handle)))
  File "/home/pi/pyusb/usb/backend/libusb1.py", line 595, in _check
    raise USBError(_strerror(ret), ret, _libusb_errno[ret])
usb.core.USBError: [Errno 13] Access denied (insufficient permissions)
```

**Comment by patkan** [comment]
The error message indicates a problem with your rights. Is your user allowed to access the USB devices? You could set up a udev rule for this.

In order to make sure that it has to do with permissions you could run `sudo python trythis2.py`. While you should not normally run scripts with root-rights, you can check this way if it is this or something else.

**Comment by DebianC** [comment]
@patkan  Thank you very much for your suggestion,and it works. I try to go on , however it occurs that Chinese characters are wrongly printed when I write like this : 
        `printer.text("你好")`  
but I get different characters printed like "挂批" 
I tried to set the encoding as #-*- coding: gbk -*- ,but got encoding problem.
And I tried to use the magic  encode like `from escpos.magicencode import Encoder`, but got no module named 'escpos.magicencode' as importError.
So I wander how to set the encoding to make my printer print correct Chinese characters.
Sorry for my English and naive questions.
Thanks again.
